observations/obs_converters/quikscat/QuikSCAT.rst

```diff
@@ -60,22 +60,27 @@ convert_L2b.f90
 
 ``convert_L2b`` converts the HDF files distributed by JPL to an obs_sequence file.
 To build ``convert_l2b`` using ``quickbuild.sh`` you will first need to build the HDF4 library.
+Note that if you are running on Derecho, you will not nned to build the HDF4 library but will
+simple need to load the module with ``module load hdf``.
 
 .. warning::
 
   To avoid conflicts with netCDF library required by DART, we recommend building HDF4 *without* 
   the HDF4 versions of the NetCDF API. 
 
-After successfully building HDF, add the appropriate library flags to your mkmf.template file. 
-Below is a snippet from an mkmf.template file used to link to both NetCDF and HDF4.   
+After successfully building HDF, add the appropriate library flags to your mkmf.template file,
+or for Derecho users, use the already available files at 
+*/DART/build_templates/mkmf.template.quikscat.intel* or 
+*/DART/build_templates/mkmf.template.quikscat.gfortran*. Below is a snippet from an
```

Comment:
I'd take out the / at the start of the path, since we want people to look for files relative to their DART directory, not the root of the filesystem. 

```suggestion
*DART/build_templates/mkmf.template.quikscat.intel* or 
*DART/build_templates/mkmf.template.quikscat.gfortran*. Below is a snippet from an
```